api/src/pcapi/core/educational/api/favorites.py

```diff
@@ -22,3 +22,12 @@ def add_offer_template_to_favorite_adage(
     )
     repository.save(favorite_offer)
     return favorite_offer
+
+
+def get_offer_to_favorite_adage(
+    redactor: educational_models.EducationalRedactor,
+) -> educational_models.CollectiveOfferEducationalRedactor:
+    favorite_offer = educational_models.CollectiveOfferEducationalRedactor.query.filter(
+        educational_models.CollectiveOfferEducationalRedactor.educationalRedactorId == redactor.id
+    ).all()
+    return favorite_offer
```

Comment:
Je ne comprends pas le nom de la fonction =/ (pourquoi offer to favorite ?)
Et... j'ai l'impression qu'elle n'est utilisée nulle part.

Reply:
En effet un code que j'ai oublié de retirer